test/setup.js

```diff
@@ -2,3 +2,6 @@ import Enzyme from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 
 Enzyme.configure({ adapter: new Adapter() })
+
+process.env.BABEL_ENV = 'test'
```

Comment:
Makes sense re: your comments — what is the workflow you’re using for testing that’s outside of `test.js`?  (Assuming just running mocha directly?)